### ISSUE TITLE
Different size of image to process

### ISSUE THREAD

**Issue by Akavan123** [steven413d/AACNN#4]
Hi,

is it possible to use different size of images and just change the numbers in main.py in lines 20 and 21 as I am comparing different CNNs and already have images in size 24x24 and would like to get result for 8x that size.

Or is it not possible to use it this way? I tried and I got some errors.

Thank you for your answer!

**Comment by steven413d** [comment]
Yes, it is possible. Please slightly modify the network part (e.g. line 68 in generator.py).
